diff --git a/node_modules/jszip/lib/compressedObject.js b/node_modules/jszip/lib/compressedObject.js
index d4a7f5e..602c6c9 100644
--- a/node_modules/jszip/lib/compressedObject.js
+++ b/node_modules/jszip/lib/compressedObject.js
@@ -4,7 +4,6 @@ var external = require("./external");
 var DataWorker = require('./stream/DataWorker');
 var DataLengthProbe = require('./stream/DataLengthProbe');
 var Crc32Probe = require('./stream/Crc32Probe');
-var DataLengthProbe = require('./stream/DataLengthProbe');
 
 /**
  * Represent a compressed object, with everything needed to decompress it.
diff --git a/node_modules/jszip/lib/load.js b/node_modules/jszip/lib/load.js
index 53cae42..357c4f8 100644
--- a/node_modules/jszip/lib/load.js
+++ b/node_modules/jszip/lib/load.js
@@ -2,7 +2,6 @@
 var utils = require('./utils');
 var external = require("./external");
 var utf8 = require('./utf8');
-var utils = require('./utils');
 var ZipEntries = require('./zipEntries');
 var Crc32Probe = require('./stream/Crc32Probe');
 var nodejsUtils = require("./nodejsUtils");
